fbpic/fields/cuda_methods.py

```diff
@@ -137,29 +137,41 @@ def cuda_divide_vector_by_volume( mode0r, mode1r, mode0t, mode1t,
 # Methods of the SpectralGrid object
 # -----------------------------------
```

Comment:
For conciseness, I removed some of the docstring.
Unfortunately, github shows this change as mixed up with the body of the function :/